### PR TITLE
[#139826733] Document restore from latest snapshot feature

### PR DESCRIPTION
**Pull request by keymon**
[#39826733 Document "restore from latest snapshot feature"](https://www.pivotaltracker.com/story/show/139826733)

What?
----

In the story [#139361587](https://www.pivotaltracker.com/n/projects/1275640/stories/139361587) we added the feature to restore postgresql DBs from the latest snapshot of a existing postgresql service.

We need to add this to the tenant manual.

It works as described in this PR: https://github.com/alphagov/paas-cf/pull/784

How to review?
--------------

Read it ;)

Who?
---

Anyone but @keymon

### CODE REVIEW

**Comment by benh-gds** [comment]
Have reviewed and rewritten for style. Have not yet tested SSH tunnelling instructions.

**Comment by dcarley** [comment]
Anyone in the team can pick up the remainder of the testing.

**Comment by saliceti** [comment]
Tested, works as expected.
The doc is clear, merging.

File: source/documentation/troubleshooting/ssh.md [hunk 1]
```diff
@@ -64,23 +64,100 @@ buildpack: staticfile_buildpack
...
+        ...
+    ```
+
+    You will need to know:
+
+    + the remote host, displayed as `"host":`
+    + the remote port, displayed as `"port"`.
+    + the PostgreSQL username, displayed as `"username":`
+    + the PostgreSQL password, displayed as `password:`
+    + the name of the database, displayed as `name:`
```
Comment (dcarley) [inline source/documentation/troubleshooting/ssh.md]: Should these be an unordered list?

At the moment they all render within a codeblock: https://docs.cloud.service.gov.uk/#creating-tcp-tunnels-with-ssh

Comment (dcarley) [inline source/documentation/troubleshooting/ssh.md]: Addressed in alphagov/paas-tech-docs#23

**Pull request merged**